roles/ansible-keylime-tpm20/tasks/ibm-tpm.yml

```diff
@@ -49,6 +49,9 @@
     dest: /etc/systemd/system/ima_emulator.service
     mode: "0644"
 
+- name: Run tpm2_startup -c
```

Comment:
I think it would be more logical to put this after the tpm_emulator has been started. Also, let's go with a better name that isn't just a "Run X" where X is the command being run. The name is a good place to explain what and why something is being run. Maybe something like "Initialize the vTPM device" or something like that.